### ISSUE TITLE
Problem: README is not user-friendly enough 

### ISSUE THREAD

**Issue by c-rack** [c-rack/quantum-elixir#45]
Currently, the structure of the README is as following:
- Setup
- Usage
  - Named Jobs
  - Nodes
  - Timezone
  - Crontab format
  - Special expressions
  - Override default settings
- Contribution
  - tl;dr
- License

I would like to:
- ~~add a ToC for quick access~~
- add a "General concepts" section and put sections about "Job", "Task" and "Cron expression" there
- split the usage section into "static configuration" and "runtime configuration" (naming is just an example)

So it would become (subject to discussion):
- ToC
- Setup
- General concepts
  - Job
    - Named Job
    - Nodes
  - Cron expression
    - Special expressions
    - Timezone
  - Task _(i.e. different ways to reference a function)_
- Usage
  - Static configuration
  - Runtime configuration
  - Override default settings
- Contribution
  - tl;dr
- License

Comments? Thoughts?

**Comment by theoretick** [comment]
Sounds great to me :+1: love a Readme w/ a TOC

**Comment by maennchen** [comment]
Could we replace this issue with #162?

TOC etc. will automatically generated by ExDoc.

**Comment by maennchen** [comment]
Replaced by #162
